src/core/chart-group-types.ts

```diff
@@ -0,0 +1,23 @@
+export interface IMinimalChart {
+    render (): void;
+
+    redraw (): void;
+
+    filterAll (): void;
+
+    focus? (): void;
+}
```

Comment:
Just a style nit, but do interfaces need to be double spaced? The gaps help readability when methods have content, but are they helpful if there are just one line declarations?

Reply:
Changed in https://github.com/dc-js/dc.js/pull/1729/commits/395967eb7773a95d4efa6e34b959a33333fe9e39